logging/bin/deploy_logging_open.sh

```diff
@@ -21,7 +21,7 @@ LOG_KB_TLS_ENABLE=${LOG_KB_TLS_ENABLE:-false}
 LOG_NODE_PLACEMENT_ENABLE=${LOG_NODE_PLACEMENT_ENABLE:-${NODE_PLACEMENT_ENABLE:-false}}
 
 source bin/tls-include.sh
-verify_cert_manager
+verify_cert_manager $LOG_NS es-transport es-rest es-admin kibana
 
```

Comment:
Hmmm....as part of the work being done in the logdeploysplit branch this file (deploy_logging_open.sh) will change quite a bit and this call no longer exists.  Looking at the branch now, I see that we're not calling this function at all right now.  Since those changes aren't pushed yet, I guess this change is "right"...we'll just need to be careful about the order in which the PRs are merged and add a comparable call into the right place on the other branch if this PR is merged first.

Reply:
right...if you're going to remove the check and do it your own way, let this change go through, then your PR will just modify the file again to remove it. Shouldn't be a problem.